AEPMessaging/Tests/E2EFunctionalTests/E2EFunctionalTests.swift

```diff
@@ -27,40 +27,41 @@ class E2EFunctionalTests: XCTestCase, AnyCodableAsserts {
     var currentMessage: Message?
     let asyncTimeout: TimeInterval = 30
     let appScope = "mobileapp://com.adobe.ajoinbounde2etestsonly"
+    let cbeScope = "mobileapp://com.adobe.ajoinbounde2etestsonly/cbeJson"
+    let cardScope = "mobileapp://com.adobe.ajoinbounde2etestsonly/cards/ms"
     var mockCache: MockCache!
     var mockRuntime: TestableExtensionRuntime!
-
-    override class func setUp() {
-        // before all
-        initializeSdk()
-    }
-        
-    override class func tearDown() {
-        // after all
+            
+    class override func setUp() {
```

Comment:
Why do we need two `setUp` functions?

Reply:
the `class` function runs one time before the whole suite of test cases runs, the other one runs before every test